src/master/main.c

```diff
@@ -26,6 +26,9 @@
 #include "service-process.h"
 #include "service-log.h"
 #include "dovecot-version.h"
+#ifdef HAVE_SYSTEMD
+#include "sd-daemon.h"
```

Comment:
For coding style consistency would be nice if this has two space indentation: `#  include`

Reply:
Of course github split the line exactly at the wrong place, at least in my browser.. To clarify:

```
#  include "sd-daemon.h"
```

Reply:
done